analyses/cellranger-analysis/waiter.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+#BSUB -P run_cellranger
+#BSUB -J waiter
+#BSUB -q standard
+#BSUB -n 1
+#BSUB -R "rusage[mem=2500]"
+#BSUB -o waiter.out
+#BSUB -e waiter.err
+#BSUB -cwd "/research/dept/dnb/core_operations/Bioinformatics/achroni/GitHub/sc-rna-seq-snap/analyses/cellranger-analysis"
+
+
+queue="standard"
+prefix="/research/dept/dnb/core_operations/Bioinformatics/achroni/GitHub/sc-rna-seq-snap/analyses/cellranger-analysis"
+
+# Function to check if there are any running jobs with the title pattern `ID.DST<some number here>`
+check_jobs() {
+    # Query bjobs and filter for jobs with titles matching the pattern
+    bjobs_output=$(bjobs | grep "DST[0-9]*")
+
+    # If the output is empty, there are no matching jobs
+    if [ -z "$bjobs_output" ]; then
+        return 0  # No matching jobs
+    else
+        return 1  # Matching jobs found
+    fi
+}
+
+# Loop until no matching jobs are found
+while true; do
+    if check_jobs; then
+        echo "$(date +"%D"): $(date +"%T"): No matching jobs found. Submitting the job..." >> waiter.log
+        # Submit your job here
+        bsub -P summarize_CellRanger -q ${queue} -n 1 -R "rusage[mem=2GB]" -R "span[hosts=1]" -J j2 -o j2.out -e j1.err ${prefix}/j2.bsub
```

Comment:
Small typo here, should be -e j2.err instead of j1.err

Reply:
Fixed here 0d2bbfb77d03a443469c1e00b571f28bc1f81996